File: Assignmentno21/task_1.java
import java.util.*;
class Logic 
{
    void Display(int iValue)
    {
        int iCnt = 0;
        int iCountEven =0;
        int iCountOdd = 0;
        for(iCnt = 1; iCnt<=iValue;iCnt++)
        {
            if((iCnt%2)==0)
            {
              iCountEven++;
            }
            else
            {
             iCountOdd++;   
            }
        }
        System.out.println("The Even numbers is"+iCountEven);
         System.out.println("The Even numbers is"+iCountOdd);

      
        
    }
}
class task_1

{
    public static void main(String A[])
    {
        int iNo = 0;
        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();

        System.out.println("Enter the number");
        iNo = sobj.nextInt();
        lobj.Display(iNo);


    }
}

File: Assignmentno9/task_1.c
#include<stdio.h>

void Display(int Num)
{
    int iCnt = 0;
    for(iCnt=0;iCnt<Num;iCnt++)
    {
        printf("*\t",iCnt);
    }
    for ( iCnt = 0; iCnt < Num; iCnt++)
    {
        printf("#\t",iCnt);
    }
    
}

int main()
{
    int iNo =0;

    printf("Enter the number :\n");
    scanf("%d",&iNo);
    Display(iNo);
    return 0;
}